dashboard/app/models/pd/payment_term.rb

```diff
@@ -73,4 +75,18 @@ def sufficient_contract_terms
       errors.add(:base, 'Must have either per attendee payment or fixed payment')
     end
   end
+
+  def truncate_previous_term
+    # When we create a new payment term, see if there are any payment terms for the exact
+    # criteria. If there is one, truncate it.
+    old_payment_term = Pd::PaymentTerm.find_by(
+      regional_partner: regional_partner,
+      course: course,
+      subject: subject
+    )
+
+    if old_payment_term
+      old_payment_term.update(end_date: start_date)
+    end
```

Comment:
This doesn't seem right. Wouldn't we only want to truncate terms that overlap this new term?

Say I have 2 terms for the same regional partner, course, & subject:
 1. June 1 - July 1
 2. July 1 - Aug 1
And we add a new one for Aug 1 - Sept 1. This will change both 1 and 2 above to end at Aug 1, resulting in ranges:
 1. June 1 - Aug 1
 2. July 1 - Aug 1

It seems in this case we shouldn't modify any of them.